slowmovie.py

```diff
@@ -184,6 +213,9 @@ def estimate_runtime(delay, increment, frames, output="guess"):
 parser.add_argument("-c", "--contrast", default=1.0, type=float, help="adjust image contrast (default: %(default)s)")
 parser.add_argument("-l", "--loop", action="store_true", help="loop a single video; otherwise play through the files in the videos directory")
 parser.add_argument("-o", "--loglevel", default="INFO", type=str.upper, choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"], help="minimum importance-level of messages displayed and saved to the logfile (default: %(default)s)")
+group = parser.add_mutually_exclusive_group()
+group.add_argument("-S", "--subtitles", action="store_true", help="Display SRT subtitles")
+group.add_argument("-t", "--timecode", action="store_true", help="Display video timecode")
```

Comment:
Should this be named `textOverlayGroup` or something more specific?

```suggestion
textOverlayGroup = parser.add_mutually_exclusive_group()
textOverlayGroup.add_argument("-S", "--subtitles", action="store_true", help="Display SRT subtitles")
textOverlayGroup.add_argument("-t", "--timecode", action="store_true", help="Display video timecode")
```